### PR TITLE
Update outdated images

### PR DESCRIPTION
**Pull request by Bermiin**

### CODE REVIEW

File: static/assets/enable_all_intents.mp4
Comment (granny) [inline static/assets/enable_all_intents.mp4:1]: Is this meant to be unused?

Comment (Bermiin) [reply static/assets/enable_all_intents.mp4:1]: Don't really know where are you seeing that its unused, but it should be used. When I localhost the docs, I can see that all media works just fine, including the intents video

Comment (granny) [reply static/assets/enable_all_intents.mp4:1]: Oooh I got confused by GitHub's UI. No worries

**Review by granny — APPROVED** [review]

**Comment by granny** [comment]
Thank you for your PR :)

**Pull request merged**